build.zig

```diff
@@ -448,18 +387,101 @@ const Paths = struct {
     bin: []const u8,
 };
 
-fn getPaths(sdk: *Sdk, target_local: std.Build.ResolvedTarget) error{ MissingTarget, FileNotFound, InvalidJson, InvalidTarget }!Paths {
-    const json_data = std.fs.cwd().readFileAlloc(sdk.build.allocator, sdk.config_path, 1 << 20) catch |err| switch (err) {
-        error.FileNotFound => return error.FileNotFound,
-        else => |e| @panic(@errorName(e)),
+const GetPathsError = error{
+    FileNotFound,
+    InvalidJson,
+    InvalidTarget,
+    MissingTarget,
+};
+
+fn printPathsErrorMessage(sdk: *Sdk, config_path: []const u8, target_local: std.Build.ResolvedTarget, err: GetPathsError, library: Library) !void {
+    const writer = std.io.getStdErr().writer();
+    const target_name = try tripleName(sdk.build.allocator, target_local);
+    defer sdk.build.allocator.free(target_name);
+
+    const lib_name = switch (library) {
+        .SDL2 => "SDL2",
+        .SDL2_ttf => "SDL2_ttf",
+    };
+
+    const download_url = switch (library) {
+        .SDL2 => "https://github.com/libsdl-org/SDL/releases",
```

Comment:
Note, I changed these links point to GitHub instead, as the latest releases are more easily downloadable from here.